examples/seismic/viscoacoustic/operators.py

```diff
@@ -127,6 +127,8 @@ def sls_2nd_order(model, geometry, p, **kwargs):
     ----------
     p : TimeFunction
         Pressure field.
+    r : TimeFunction
+        Memory variable.
```

Comment:
"Memory"?

Reply:
yes, this is just the name of a variable in the PDE. 
There is no relationship with the computational part.